src/main/java/salesforce/ui/pages/opportunity/newopportunity/AbstractNewOpportunity.java

```diff
@@ -0,0 +1,367 @@
+     *
+     * @param deliveryStatus value.
+     */
+    public abstract void selectDeliveryStatus(String deliveryStatus);
+
+    /**
+     * Sets description value.
+     *
+     * @param description string.
+     */
+    public abstract void setDescriptionTextarea(String description);
+
+    /**
+     * Clicks on save button to create the opportunity.
+     */
+    public abstract void clickOnSaveButton();
+
+    /**
+     * Clicks on cancel button to create the opportunity.
+     */
+    public abstract void clickOnCancelButton();
+
+    /**
+     * Gets opportunity name.
+     *
+     * @return opportunity name value.
+     */
+    public abstract String getOpportunityName();
+
+    /**
+     * Gets account name.
+     *
+     * @return account name value.
+     */
+    public abstract String geAccountName();
+
+    /**
+     * Gets type.
+     *
+     * @return type value.
+     */
+    public abstract String getType();
+
+    /**
+     * Gets lead.
+     *
+     * @return lead value.
+     */
+    public abstract String getLead();
+
+    /**
+     * Gets amount.
+     *
+     * @return amount value.
+     */
+    public abstract String getAmount();
+
+    /**
+     * Gets close date.
+     *
+     * @return close data value.
+     */
+    public abstract String getCloseDate();
+
+    /**
+     * Gets next step value.
+     *
+     * @return next step.
+     */
+    public abstract String getNextStep();
+
+    /**
+     * Gets stage.
+     *
+     * @return stage value.
+     */
+    public abstract String getStage();
+
+    /**
+     * Gets probability.
+     *
+     * @return probability value.
+     */
+    public abstract String getProbability();
+
+    /**
+     * Gets campaign value.
+     *
+     * @return campaign name;
+     */
+    public abstract String getCampaign();
+
+    /**
+     * Gets order number value.
+     *
+     * @return order number;
+     */
+    public abstract String getOrderNumber();
+
+    /**
+     * Gets current generator value.
+     *
+     * @return generator value.
+     */
+    public abstract String getCurrentGenerator();
+
+    /**
+     * Gets tracking number.
+     *
+     * @return tracking number value.
+     */
+    public abstract String getTrackingNumber();
+
+    /**
+     * Gets main competitor value.
+     *
+     * @return main competitor text.
+     */
+    public abstract String getMainCompetitor();
+
+    /**
+     * Gets delivery status value.
+     *
+     * @return status text.
+     */
+    public abstract String getDeliveryStatus();
+
+    /**
+     * Gets description value.
+     *
+     * @return description text.
+     */
+    public abstract String getDescription();
+
+    /**
+     * Adds opportunity values.
+     *
+     * @param opportunity object.
+     * @param fields list.
+     * @return a new instance of opportunity page.
+     */
+    public AbstractOpportunityPage addOpportunityInformation(final Opportunity opportunity, final Set<String> fields) {
```

Comment:
in this part could be setOpportunityInformation

Reply:
It is okay